scripts/publish.js

```diff
@@ -14,10 +14,15 @@ const { version } = json;
 console.log(`Publishing version: ${version}`);
 
 // Publish command to post to npm - must be run in the same directory as the gestalt package
-shell.cd(path.join(__dirname, '..', 'packages', 'gestalt'));
-shell.exec(
-  `yarn publish --registry=https://registry.npmjs.org --no-git-tag-version --new-version ${version}`
-);
+// `yarn publish` publishes the package defined by the package.json in the current directory.
+const packages = ['gestalt', 'gestalt-datepicker'];
```

Comment:
I modified this file but I think the release.yml is the file that executes the publishing currently.